src/components/UserProfile/VolunteeringTimeTab/VolunteeringTimeTab.jsx

```diff
@@ -51,6 +52,7 @@ const EndDate = props => {
       }}
       placeholder="End Date"
       invalid={!props.canEdit}
+      min={moment(props.userProfile.createdDate).format('YYYY-MM-DD')}
```

Comment:
Do we need to  verify that the property exists?

```suggestion
      min={props.userProfile.createdDate ? moment(props.userProfile.createdDate).format('YYYY-MM-DD') : ''}
```

Reply:
Nice catch, The created date should always exist (from the API) but shouldn't hurt to add this check in-case something changes in the future.